inductiva/storage/README.md

```diff
@@ -31,6 +31,12 @@ This function generates a table displaying the storage directories:
 | 1234567| 97.59 KiB  | 29 Sep, 14:13:24|
 | 123    | 0 B        | 29 Sep, 14:13:29|
 ```
+To examine the contents of a specific folder, execute: 
+
+```python
+import inductiva
+inductiva.storage.listdir(path = "1234/", max_results=10, order_by="size", sort_order="desc")
```

Comment:
Do they need to pass the "/" at the end?

Reply:
No, if the user doesn't put it We will add it in the backend.

Reply:
Have we tested that the user can't go up using ".."?

Reply:
![image](https://github.com/inductiva/inductiva/assets/114397668/322e7124-98c7-4129-a11f-7ba9c1470da6)